pyfolio/pos.py

```diff
@@ -57,13 +57,14 @@ def get_long_short_pos(positions):
 
     pos_wo_cash = positions.drop('cash', axis=1)
     longs = pos_wo_cash[pos_wo_cash > 0].sum(axis=1).fillna(0)
-    shorts = pos_wo_cash[pos_wo_cash < 0].abs().sum(axis=1).fillna(0)
+    shorts = pos_wo_cash[pos_wo_cash < 0].sum(axis=1).fillna(0)
```

Comment:
if `shorts` is a negative number, net liquidation should be `longs + shorts + cash`